cli/src/pcluster/validators/cluster_validators.py

```diff
@@ -754,6 +756,24 @@ def _validate(self, mount_dir: str):
             )
 
 
+class DeletionPolicyValidator(Validator):
+    """Print warning message when deletion policy is set to Delete or Retain."""
+
+    def _validate(self, deletion_policy: str, name: str):
```

Comment:
Generally speaking, I think we should not have warning message in both cases, but keep the one that we consider most important. I would vote for avoiding the message for the behaviour that we consider the default.

Reply:
the other approach I agree with is to have INFO messages in both cases.

Reply:
Changed to INFO level